templates/confidential-computing/index.html

```diff
@@ -0,0 +1,287 @@
+{% extends "templates/base.html" %}
```

Comment:
yes, I noticed that and honestly didn't understand why. Is there a reason why you do so?

Reply:
There may be things we set in the bubble base that don't necessarily have anything to do with site as a whole, eg: if all pages share the same form you can add that marketo div at the bottom to the bubble base instead of having to add it to each individual page. We also tend to add the drive folder (as opposed to individual docs) for the bubble in the base. That said I've never read any documentation on why we do this, the above is just what I know from experience 😅

Reply:
makes sense if there are several pages in the bubble. In my case there is only one, but for the future it can be done like that indeed.

Reply:
addressed in https://github.com/canonical/ubuntu.com/pull/13176